api/wallet.go

```diff
@@ -207,9 +207,9 @@ func (a *API) walletCreateHandler(msg *bearerstdapi.BearerStandardAPIdata, ctx *
 
 	stx := models.SignedTx{}
 	stx.Tx, err = proto.Marshal(&models.Tx{
-		Payload: &models.Tx_SetAccountInfo{
-			SetAccountInfo: &models.SetAccountInfoTx{
-				Txtype:        models.TxType_SET_ACCOUNT_INFO,
+		Payload: &models.Tx_SetAccount{
+			SetAccount: &models.SetAccountTx{
+				Txtype:        models.TxType_SET_ACCOUNT_INFO_URI,
 				Nonce:         0,
 				InfoURI:       "none",
```

Comment:
Yes it CAN be empty, but this method needs to be refactored anyways for adding delegates too on the account bootstrap. I was planning to do it when tacking the client and the API with the tests.

I made it empty btw